### PR TITLE
fix: remove proxy-rewrite plugin out of create plugin template config view

### LINKED ISSUE #1719: plugin "proxy-rewrite" found in plugin-config detail page and not found in route detail page
## Issue description
 plugin "proxy-rewrite" found in plugin-config detail page and not found in route detail page

## Expected behavior
 plugin "proxy-rewrite" both found or not found in plugin-config detail page and route detail page

## How to Reproduce

When creating the route, the "proxy-rewrite" plugin was not found ；

When creating the  plugin-config, the "proxy-rewrite" plugin was found ；

## Screenshots

image

## Environment

image

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

While checking #1719 , another bug happened, `proxy-rewrite` plugin can still be configed in `plugin template` page.
Remove `proxy-rewrite` out of  create plugin template config step2 , just keep the same as the create route step3 does.

**Related issues**

fix/resolve None

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

File: web/src/pages/PluginTemplate/Create.tsx [hunk 1]
```diff
@@ -108,7 +108,12 @@ const Page: React.FC = (props) => {
 
           {step === 1 && <Step1 form={form1} />}
           {step === 2 && (
-            <PluginPage initialData={plugins} onChange={setPlugins} schemaType="route" />
+            <PluginPage
```
Comment (jbampton) [inline web/src/pages/PluginTemplate/Create.tsx:111]: I have never done `.tsx` before, but when I try to do HTML attributes I like to order them alphabetically.

Suggested change:

```
<PluginPage
    initialData={plugins}
    onChange={setPlugins}
    referPage="route"
    schemaType="route"
    />
```

**Review by membphis — APPROVED** [review]

File: web/src/pages/PluginTemplate/Create.tsx (hunk 1 above)
Comment (juzhiyuan) [reply web/src/pages/PluginTemplate/Create.tsx:111]: yep

Comment (liuxiran) [reply web/src/pages/PluginTemplate/Create.tsx:111]: thanks for your useful advice, I have updated the code, PTAL when you are free. @justinmclean

**Review by imjoey — APPROVED** [review]
/lgtm

Thanks, @liuxiran.

**Pull request merged**